internal/profile/static.go

```diff
@@ -124,7 +123,8 @@ func newPackageRegistryConfig(_ string, profilePath string) (*simpleFile, error)
 }
 
 // PackageRegistryBaseImage is the base Docker image of the Elastic Package Registry.
-const PackageRegistryBaseImage = "docker.elastic.co/package-registry/distribution:snapshot"
+// commit SHA of enabled proxy mode (technical preview feature)
+const PackageRegistryBaseImage = "docker.elastic.co/package-registry/package-registry:14455fcb7e415d6b4aed48e00d2289718f056162"
```

Comment:
Do you want to wait to have a released version with this?

Reply:
Yes, as I'm afraid that we may need to add more fixes, and going with the standard release will slow it down. I will replace it with a proper release after some time.